### PR TITLE
feat(sdk): add Bedrock Alpha Testnet contracts

### PR DESCRIPTION
**Pull request by smartcontracts**
**Description**
Adds contract addresses for the Bedrock Alpha testnet.

### CODE REVIEW

File: packages/sdk/src/utils/chain-constants.ts [hunk 1]
```diff
@@ -146,6 +146,22 @@ export const CONTRACT_ADDRESSES: {
     },
     l2: DEFAULT_L2_CONTRACT_ADDRESSES,
   },
+  [L2ChainID.OPTIMISM_BEDROCK_ALPHA_TESTNET]: {
+    l1: {
+      AddressManager: '0xb4e08DcE1F323608229265c9d4125E22a4B9dbAF' as const,
```
Comment (tynes) [inline packages/sdk/src/utils/chain-constants.ts:151]: https://goerli.etherscan.io/address/0xb4e08DcE1F323608229265c9d4125E22a4B9dbAF

**Review by tynes — APPROVED** [review]

**Review by mslipper — APPROVED** [review]

**Comment by tynes** [comment]
@smartcontracts Any reason this is still a draft?

**Comment by smartcontracts** [comment]
> @smartcontracts Any reason this is still a draft?

Did not get confirmation that this worked, and spotted a bug. Going to get confirmation from Ori in a sec.

**Comment by smartcontracts** [comment]
Got confirmation from Ori that this is working

**Pull request merged**